ncm-metaconfig/src/main/metaconfig/cumulus/pan/ports.pan

```diff
@@ -0,0 +1,9 @@
+unique template metaconfig/cumulus/ports;
+
+include 'metaconfig/cumulus/schema';
+
+bind "/software/components/metaconfig/services/{/etc/cumulus/ports.conf}/contents" = cumulus_ports;
+
+prefix "/software/components/metaconfig/services/{/etc/cumulus/ports.conf}";
+"module" = "cumulus/ports";
+# daemons: update-ports
```

Comment:
Is this copypasta or useful code that got commented out?

Reply:
possibly useful code. i haven't run this yet on a cumulus switch
i ostly do `ifreload -a` after if copied the files manually in place